### ISSUE TITLE
Debug slight inconsistencies in percentile calculations

### ISSUE THREAD

**Issue by annapowellsmith** [ebmdatalab/openprescribing#120]
While researching BigQuery's performance at calculating deciles on the fly, I noticed some disparities between the results returned by BigQuery (and verified independently by me as correct) and the percentiles in our calculated measures. 

I used cerazette measures for practices for January 2016 as my test data. The 90th percentile returned by BigQuery is 0.60389 and my own calculations are in the region of 0.6048. However the 90th percentile in our calculated measures is 0.59918. Other deciles are off by a similar small amount. 

I also noticed that for [this practice](https://openprescribing.net/practice/B81080/measures/), the Cerazette measure for January 2016 is 100%, but the practice is listed as being at the 97th percentile. This is because [we're using `method='min'` with scipy's `rankdata`](https://github.com/ebmdatalab/openprescribing/blob/master/openprescribing/frontend/management/commands/import_measures.py#L339-L340), but we should review whether it's the best way to do things, and whether it's affecting the calculations above. 

Debug this and correct any incorrect values.

**Comment by annapowellsmith** [comment]
OK, I think this is because we're using a different set of practices - in our import script we explicitly exclude non-standard practices and practices that were closed or unopened on the date in question, whereas the BigQuery query I used included all practices (and we know that some non-standard practices do prescribe, and also that there can be a lag between a practice closing and prescriptions being processed). 

It would still be good to verify using BigQuery though, I'll see if I can do this without it getting too complicated.

**Comment by annapowellsmith** [comment]
I'm closing this as not significant, see my [debugging notes](https://docs.google.com/document/d/11liywINtoCLT6S9wghzCl5ry8kNXj3waoG_uc864ZWI/edit). 

Basically I found that the margin of error is tiny, and probably to do with [there being no standard quantile calculation method](https://en.wikipedia.org/wiki/Quartile#Computing_methods).

After some thought, I also decided that using the `min` method in rankdata is the best approach, for two reasons:

1. For most of our measures, higher is worse, so we're effectively giving organisations the benefit of the doubt where available, which feels like the right thing to do
2. The modal measure will be 0, not 100, so using `min` will be confusing less often

However, we can review this if it does cause usability problems.
